### PR TITLE
#31 support Validate pages , `./gradlew bootRun -Pargs=--site=1,--pro…

### PR DESCRIPTION
**Pull request by jiangliwu**
#31  `./gradlew bootRun -Pargs=--site=1,--proc=validate-pages`

### CODE REVIEW

File: crawler-service/src/main/java/com/topcoder/productsearch/validatePages/service/ValidatePagesService.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package com.topcoder.productsearch.validatePages.service;
```
Comment (ykohata) [inline crawler-service/src/main/java/com/topcoder/productsearch/validatePages/service/ValidatePagesService.java]: @jiangliwu - could you move this class `ValidatePagesService` to the package `com.topcoder.productsearch.cleaner.service`?

Comment (jiangliwu) [reply crawler-service/src/main/java/com/topcoder/productsearch/validatePages/service/ValidatePagesService.java]: @ykohata sure, updated

**Comment by azhar-cograph** [comment]
Looks good.  The usage information can be added to the ProcessLauncher.class.  But we can add this ourselves. 

      logger.info("usage : ./gradlew bootRun -Pargs=--site=1,--proc=validate-pages");

**Comment by azhar-cograph** [comment]
@jiangliwu  How are you only reading in records where the 'deleted' flag is false? which is a requirement.  Maybe I can't find it in the code.

**Comment by jiangliwu** [comment]
@azhar-cograph good point, i missing that, let me add this

**Comment by jiangliwu** [comment]
@azhar-cograph updated

**Pull request merged**